### DISCUSSION TITLE
Slow Server Frames Error PLEASE HELP

### DISCUSSION THREAD

**Discussion by ergin85** [kus/cs2-modded-server#157] [Q&A]
Slow server frames error. My server details in the photo. Can you help me ?

**Comment by ergin85** [comment]
Server details : https://ibb.co/nQjfpBf

**Comment by YD-S** [comment]
same problem here, can you execute a mode and check the console

**Reply by ergin85** [reply]
https://drive.google.com/file/d/1w00aFj92CLRiM0x34RouAhSu_IDyq7mU/view?usp=sharing Here you are.

**Comment by YD-S** [comment]
i have a problem where like i have a lot of lag, when i play openprefireprac i cant even move properly, can you execute prefire and record a longer vid

**Reply by ergin85** [reply]
I opened the this server for retakes. I deleted other mods command like !aim etc.

**Reply by YD-S** [reply]
> when you open the .prac mode, when you throw smokes, it takes ages to throw the smoke like 2-3 seconds after throwing

**Reply by ergin85** [reply]
What we can do for this issue.

**Comment by YD-S** [comment]
i am going to open a issue, ask kus, and wait
